Frontend-v1-Original/components/ssVotes/ssVotes.tsx

```diff
@@ -276,21 +276,36 @@ export default function ssVotes() {
     [gauges, search]
   );
 
+  const bribesAccrued = filteredGauges
+    .reduce(
+      (
+        acc: number,
+        row: {
+          gauge: {
+            tbv: number;
+          };
+        }
+      ) => acc + row.gauge.tbv,
+      0
+    )
+    .toFixed(2);
```

Comment:
```suggestion
    .toFixed(2)
    .toLocaleString();
```
Might be a good idea to format large numbers with `toLocaleString`?

E.g., showing `$20,032.8` instead of `$20032.80`.

Reply:
@oxSaturn sure. to be consistent you can import function called formatCurrency
`import { formatCurrency } from "../../utils/utils";`
 and do something like 
`...{$formatCurrency(bribesAccrued)}...`